_data/sidebar_doc.yml

```diff
@@ -34,10 +34,36 @@ entries:
                 - title: Use the Built-in SQL Client
                   url: /use-the-built-in-sql-client.html
 
-        - title: Build a Test App
-          url: /build-a-test-app.html
+            - title: Connect an App
```

Comment:
"Connect an App" is confusing to me. Is this about building an app, or connecting from an app to CockroachDB?

How about "Build an App"?